client/src/endToEndManualTest/java/io/obswebsocket/community/client/test/AbstractObsE2ETest.java

```diff
@@ -63,15 +58,10 @@ protected static void connectToObs() {
           System.out.println("An error occurred: " + reasonThrowable.getReason());
           reasonThrowable.getThrowable().printStackTrace();
         }))
+        .onCommunicatorError(e -> fail("Unable to connect", e))
+        .onControllerError(e -> fail("Error", e))
```

Comment:
`onControllerError` twice?

Reply:
Communication/Controller error

Reply:
Removed the duplicate. It now uses `fail` which might not actually fail the test, but the original one also just logged an error.